nucypher/blockchain/eth/interfaces.py

```diff
@@ -102,11 +112,13 @@ class NotEnoughConfirmations(InterfaceError):
         pass
 
     def __init__(self,
-                 poa: bool = True,
+                 emitter = None,
```

Comment:
Why does it take an emitter? Doesn't seem to be used anywhere.

Reply:
#1754